src/core/chat.ts

```diff
@@ -92,13 +91,23 @@ export class ChatClient {
     return this._clientOptions;
   }
 
+  /**
+   * Adds additional agent information to the client.
+   * Used internally to add React-specific agent information.
+   * @param agent - The agent to add.
+   * @internal
+   */
+  public addAdditionalAgent(agent: string): void {
```

Comment:
Perhaps as React is the only valid option here, we should have `addReactAgent`?

Reply:
If this is the case, I will remove the input arg since it will always be the same.